src/physics/exact_solutions/exact_solution.cpp

```diff
@@ -0,0 +1,89 @@
+#include <deal.II/base/function.h>
+#include "exact_solution.h"
+
+namespace PHiLiP {
+
+// ========================================================
+// ZERO -- Returns zero everywhere; used a placeholder when no exact solution is defined.
+// ========================================================
+template <int dim, int nstate, typename real>
+ExactSolutionFunction_Zero<dim,nstate,real>
+::ExactSolutionFunction_Zero(double time_compare)
+        : ExactSolutionFunction<dim,nstate,real>()
+{
+    t = time_compare;
```

Comment:
Can place in the member initializer list, as `t(time_compare)`

Reply:
Good point, that is a nicer way to initialize. I will change that in the next commit.